src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -461,6 +467,32 @@ private void verifyUpload(Exporter3Configuration ex3Config, String uploadId, Str
 
         long participantModifiedOn = Long.parseLong(rowMap.get("modifiedOn"));
         assertTrue(participantModifiedOn > oneHourAgo.getMillis());
+
+        // Verify the record in Bridge.
+        ForWorkersApi workersApi = adminDeveloperWorker.getClient(ForWorkersApi.class);
+        HealthDataRecordEx3 record = workersApi.getRecordEx3(IntegTestUtils.TEST_APP_ID, uploadId).execute().body();
+        assertTrue(record.isExported());
+        assertTrue(record.getExportedOn().isAfter(oneHourAgo));
+
+        // Verify the presigned download url for a health record is generated and contains the data expected.
+        String hostUrl = user.getClientManager().getHostUrl();
+        String url = "/v3/participants/self/exporter3/healthdata/" + uploadId + "?download=true";
+
+        HttpResponse response = Request.Get(hostUrl + url)
+                .setHeader("Bridge-Session", user.getSession().getSessionToken())
+                .setHeader("Accept-Language", "en")
+                .execute().returnResponse();
+        assertEquals(200, response.getStatusLine().getStatusCode());
+
+        JsonNode node = new ObjectMapper().readTree(EntityUtils.toString(response.getEntity()));
+        HttpResponse responseForPresignedUrl =Request.Get(node.findValue("downloadUrl").asText())
+                .setHeader("Bridge-Session", user.getSession().getSessionToken())
+                .setHeader("Accept-Language", "en")
```

Comment:
The download URL is in S3, not Bridge, so we don't need to pass in Bridge-Session or Accept-Language.